pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/PolicyHierarchyValue.java

```diff
@@ -53,15 +56,21 @@ public void updateNamespaceValue(T namespaceValue) {
         updateValue();
     }
 
-    public void updateTopicValue(T topicValue) {
-        this.topicValue = topicValue;
+    public void updateTopicValue(T topicValue, boolean isGlobalPolicy) {
```

Comment:
`public void updateTopicValue(T topicValue)` should be kept for compatible.

Reply:
Added it back and added a modifier `@Deprecated`